### ISSUE TITLE
Missing source maps for maplibre-gl

### ISSUE THREAD

**Issue by phb-ig** [plotly/plotly.js#7161]
Webpack gives the following compiler warning:

```
Failed to parse source map from 'node_modules/plotly.js/dist/maplibre-gl-unminified.js.map' file: Error: ENOENT: no such file or directory, open 'node_modules/plotly.js/dist/maplibre-gl-unminified.js.map'
```

Since: 2.35.0

**Comment by junaidmzafar** [comment]
I am also seeing the same issue

**Comment by JohannSuarez** [comment]
Same here

**Comment by gvwilson** [comment]
@archmoj please dig into this - is it a configuration issue or did we miss something in the release?

**Comment by archmoj** [comment]
We will switch from `webpack` to `esbuild` in the next minor/major version.
The `esbuild` PR (#6909) is already merged into master.
I think we could close this issue. 
What do you think @birkskyum? If that's something you may want to fix (on top of [release-2.35.3](https://github.com/plotly/plotly.js/tree/release-v2.35.3) branch), I could release it in a patch.

**Comment by archmoj** [comment]
Addressed by #7204.
